### DISCUSSION TITLE
How to search for "%" Symbol

### DISCUSSION THREAD

**Discussion by dontthroworanges** [microsoft-search/pnp-modern-search#3987] [Q&A]
Is there a way to search for the % symbol? I am able to search for other symbols without issue, but searching for % doesn't seem to work. I am assuming this is because it's reserved for encoding purposes? I need to be able to find items with this symbol so I can replace it. Thanks!

**Comment by wobba** [comment]
Not as part of a query, only for managed properties with complete matching enabled. % is one of the characters being tokenized away.
